### ISSUE TITLE
Adaptive card version 1.5 does not display in teams

### ISSUE THREAD

**Issue by SkylerK-GG** [MicrosoftDocs/msteams-docs#9801]
I am currently using teams for desktop, version 1.6.00.27573, and attempting to send an adaptive card via power automate. The card I am trying to send is as follows:

```
{
    "type": "AdaptiveCard",
    "$schema": "http://adaptivecards.io/schemas/adaptive-card.json",
    "version": "1.5",
    "body": [
        {
            "type": "TextBlock",
            "text": "Test",
            "wrap": true
        }
    ]
}
```

Although the documentation states that adaptive card 1.5 is supported, all I get is a message that says "We're sorry, this card couldn't be displayed". If the version is changed to 1.4, it displays fine.

---
#### Document Details

⚠ *Do not edit this section. It is required for learn.microsoft.com ➟ GitHub issue linking.*

* ID: 3fb4394d-25b1-8752-3996-fe3a2934732d
* Version Independent ID: 6d391381-89d0-117c-27cc-00043bd26a36
* Content: [Types of cards - Teams](https://learn.microsoft.com/en-us/microsoftteams/platform/task-modules-and-cards/cards/cards-reference)
* Content Source: [msteams-platform/task-modules-and-cards/cards/cards-reference.md](https://github.com/MicrosoftDocs/msteams-docs/blob/main/msteams-platform/task-modules-and-cards/cards/cards-reference.md)
* Service: **msteams**
* GitHub Login: @surbhigupta
* Microsoft Alias: **surbhigupta**

**Comment by Meghana-MSFT** [comment]
We are able to repro this issue, we have raised a bug for the same. We will keep you posted on the updates. Thank you.

**Comment by SkylerK-GG** [comment]
Small update, same issue also occurs on android. Tested on teams version 1416/1.0.0.2023192801/1016.

Any ETA on when this will be resolved?

**Comment by Meghana-MSFT** [comment]
Unfortunately, we do not have any ETA to share. We will keep you posted on the updates. Thank you.

**Comment by Meghana-MSFT** [comment]
Apologies for the inconvenience caused to you, we do not have any ETA to share. We are following up with engineering team for updates. Thank you.

**Comment by gbassinello** [comment]
Still with this issue. Adaptive Cards 1.5 works for Channel Webhooks but not from Power Automate "Post Card to Chat or Channel" action.

**Comment by Meghana-MSFT** [comment]
The note mentions only about adaptive cards sent via bots, message extensions etc but not Power Automate. We are also checking about the status of above-mentioned bug. We will keep you posted on the updates.

**Comment by mrpaulreed** [comment]
I notice that Channel Webhooks are about to decommission. Does that mean that this bug will be fixed in time ? I have much that will fail when I am forced into Workflows if 1.5 is still not working !?!??!

**Comment by jwbeck97** [comment]
> I notice that Channel Webhooks are about to decommission. Does that mean that this bug will be fixed in time ? I have much that will fail when I am forced into Workflows if 1.5 is still not working !?!??!

Same here

**Comment by greg0rym** [comment]
This is an issue for us given we are being forced away from connectors to workflows on a short timeline.

**Comment by xolf** [comment]
> This is an issue for us given we are being forced away from connectors to workflows on a short timeline.

Same here. Workflows even didn't worked for me so I was required to upgrade to Power Automate Premium to create my own HTTP Post endpoint and send them via a flow to be limited to 1.4 version.

**Comment by mrpaulreed** [comment]
Hey @Meghana-MSFT - any further update on this considering that many features will soon cease to function once HTML is no longer available in WorkFLows. Like for example AdaptiveCardTables ( which is my killer ) ?

**Comment by Andrew-J-Larson** [comment]
> Still with this issue. Adaptive Cards 1.5 works for Channel Webhooks but not from Power Automate "Post Card to Chat or Channel" action. The [documentation](https://learn.microsoft.com/en-us/microsoftteams/platform/task-modules-and-cards/cards/cards-reference#support-for-adaptive-cards) says 1.5 is supported.

True, I've found sticking to version `1.4` is what I have to do for now. I've avoided using any HTML based formatting, and only using markdown.

**Comment by NSundararajanRTGBI** [comment]
Is this issue fixed? I'm also having same issue. With new Teams Workflow. I'm getting message "we're sorry, this card coudn't be displayed.

**Comment by FluffyMatt** [comment]
Echoing the other statements here 1.5 isn't working at all

**Comment by kalexnolasco** [comment]
Please 🥺. We need to use the 1.5 version.

**Comment by ponvthc** [comment]
Agree, this is an issue that needs to be addressed.

**Comment by mnoah66** [comment]
Any update?? I need 1.5 in order to use universal actions (which, according to documentation is only supported on 1.5)

**Comment by Andrew-J-Larson** [comment]
Not sure who needs to know at this point, but this has been fixed sometime between the last mention and now.

Every adaptive card test with version 1.5 is working for me.

I suspect this lines up with Microsoft completely stopping Teams classic from working anymore, which happened on [July 1st](https://learn.microsoft.com/en-us/microsoftteams/teams-classic-client-end-of-availability#:~:text=The%20end%20of%20support%20for%20the%20classic%20Teams%20client%20starts%20July%201%2C%202024.%20The%20end%20of%20availability%20for%20the%20classic%20Teams%20client%20starts%20July%201%2C%202025.)

I bet the original inclusion of 1.5 on the docs was supposed to mean only for the new Teams, and was likely added too long in advance before the deprecation of the classic Teams (which could only use the 1.4 version). And they couldn't just enable support for 1.5, because classic Teams also still had a web app at the time.

**Comment by Prasad-MSFT** [comment]
We checked this with version 1.5 & 1.6 and observed that the card renders fine. This bug seems to be fixed.

<img width="829" height="169" alt="Image" src="https://github.com/user-attachments/assets/67a75f56-15b2-4cf4-bd49-ccf46856a982" />
